app/presenters/vacancy_presenter.rb

```diff
@@ -75,10 +75,16 @@ def show_primary_job_role
     I18n.t("helpers.label.publishers_job_listing_job_role_form.primary_job_role_options.#{primary_job_role}")
   end
 
+  def additional_job_role(role)
+    I18n.t("helpers.label.publishers_job_listing_job_role_details_form.additional_job_roles_options.#{role}")
+  end
+
+  def public_job_roles
```

Comment:
I'm not sure what the meaning of `public` is here.

Reply:
as in public listing

Reply:
It seems to break the established convention of using 'show'?